### PR TITLE
OSX.md -> macOS.md!

### PR DESCRIPTION
**Pull request by Antoine007**

### CODE REVIEW

File: macOS.md [hunk 1]
```diff
@@ -433,19 +433,18 @@ pane, click the radio button `All controls`. This way when you get a dialog with
...
 ```bash
-# Expanding the save panel by default
-defaults write NSGlobalDomain NSNavPanelExpandedStateForSaveMode -bool true
-defaults write NSGlobalDomain PMPrintingExpandedStateForPrint -bool true
-defaults write NSGlobalDomain PMPrintingExpandedStateForPrint2 -bool true
 
-# Disable system-wide resume
-defaults write com.apple.systempreferences NSQuitAlwaysKeepsWindows -bool false
+# Don’t display the annoying prompt when quitting iTerm
+defaults write com.googlecode.iterm2 PromptOnQuit -bool false
```
Comment (ssaunier) [inline macOS.md]: iTerm2 is not part of the official Wagon setup.

File: macOS.md [hunk 2]
```diff
@@ -433,19 +433,18 @@ pane, click the radio button `All controls`. This way when you get a dialog with
...
+
+# Save screenshots to the desktop
+defaults write com.apple.screencapture location -string "${HOME}/Desktop"
```
Comment (ssaunier) [inline macOS.md]: Isn't it the default path?

File: macOS.md [hunk 3]
```diff
@@ -433,19 +433,18 @@ pane, click the radio button `All controls`. This way when you get a dialog with
 you'll be able to type `Enter` to confirm, or `Space` to choose the cancel option. If you have more than
 two options, you can use tab to circle between them.
 
-### OSX For hackers
+### macOS For hackers
 
-[Read this script](https://gist.github.com/brandonb927/3195465) and cherry-pick some stuff you think will suit you.
+[Read this script](https://github.com/mathiasbynens/dotfiles/blob/master/.macos) and cherry-pick some stuff you think will suit you.
```
Comment (ssaunier) [inline macOS.md]: Thanks, will update that in the source file (this .md file is built)

File: macOS.md (hunk 2 above)
Comment (Antoine007) [reply macOS.md]: I think it is, yes

File: macOS.md (hunk 1 above)
Comment (Antoine007) [reply macOS.md]: just a random example that was more relevent than the one that were there. Pick your favourites!

Comment (ssaunier) [reply macOS.md]: OK I'll pick the screencapture then, thanks :+1:

**Pull request closed without merging**